### PR TITLE
Add release notes for 45

### PR DESCRIPTION
**Pull request by Callisto13**
Please check that everything is there which should be. let me know if anything is missing.

### CODE REVIEW

**Comment by cPu1** [comment]
I'd like to get https://github.com/weaveworks/eksctl/pull/3544 out in this release.

**Comment by Callisto13** [comment]
> I'd like to get #3544 out in this release.

Yep just getting pieces ready to go. Can you have that merged in the next 45 mins?

File: docs/release_notes/0.45.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Release 0.45.0
+
+## Features
+
+- Get nodegroup desired capacity from autoscaling instead of cloudformation (#3515)
```
Comment (aclevername) [inline docs/release_notes/0.45.0.md:5]: this gets modified again in https://github.com/weaveworks/eksctl/pull/3543 to include min & max capacity

Comment (Callisto13) [reply docs/release_notes/0.45.0.md:5]: lol i thought 3543 looked familiar but didn't know if i was imagining it 😂 

okay when the other one is merged i will swap this line

File: docs/release_notes/0.45.0.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Release 0.45.0
+
+## Features
+
+- Use the AutoScalingGroup for getting & scaling nodegroups (#3543)
+- Make EKS 1.19 the default (#3526)
+- Support passing releaseVersion for Managed Nodegroups (#3544)
```
Comment (cPu1) [inline docs/release_notes/0.45.0.md:7]:
```suggestion
- Support multiple AMI release versions in managed node groups
```

Comment (Callisto13) [reply docs/release_notes/0.45.0.md:7]: is that accurate?

Comment (cPu1) [reply docs/release_notes/0.45.0.md:7]: Yeah, it doesn't seem accurate to me either but I was trying to use the title Mike used in the issue: https://github.com/weaveworks/eksctl/issues/3506

Comment (Callisto13) [reply docs/release_notes/0.45.0.md:7]: meh i think i like your PR title better. i feel it is less likely to confuse people

Comment (cPu1) [reply docs/release_notes/0.45.0.md:7]: I think this is better:
```
- Support specifying an AMI release versions for managed node groups (#3544)
```

Comment (Callisto13) [reply docs/release_notes/0.45.0.md:7]: oh yeh that's perfect

**Review by cPu1 — APPROVED** [review]

**Comment by lloydchang** [comment]
@Callisto13 @cPu @aclevername 

Would you kindly amend the release notes as follows?

• Add #3519 next to #3516 in Bug Fixes
• Add me to Acknowledgements so when the release is finalized, then I get a GitHub notification?

Thank you.

```
## Bug Fixes
- Support comma separated array for group flag in create iamidentitymapping (#3549)
- Add capacity-optimized-prioritized (#3516, #3519)
````

```
## Acknowledgments
Weaveworks would like to sincerely thank:
  @omnibrian, @lloydchang
```

File: docs/release_notes/0.45.0.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Improvements
+- Preserve capabilities when creating changeset (#3548)
+- Eksctl scale nodegroups supports only providing one of min/max/desired size (#3511)
+- Use built-in AMI bootstrapping scripts for unmanaged nodes (#3492)
+
+
+## Bug Fixes
+- Support comma separated array for group flag in create iamidentitymapping (#3549)
+- Add capacity-optimized-prioritized (#3516)
```
Comment (lloydchang) [inline docs/release_notes/0.45.0.md:17]:
```suggestion
- Add capacity-optimized-prioritized (#3516, #3519)
```

File: docs/release_notes/0.45.0.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Acknowledgments
+Weaveworks would like to sincerely thank:
+  @omnibrian
```
Comment (lloydchang) [inline docs/release_notes/0.45.0.md:21]:
```suggestion
  @omnibrian, @lloydchang
```

**Review by lloydchang — COMMENTED** [review]
@Callisto13 @cPu @aclevername 

Would you kindly amend the release notes as follows?

• Add #3519 next to #3516 in Bug Fixes
• Add me to Acknowledgements so when the release is finalized, then I get a GitHub notification?

Thank you.

```
## Bug Fixes
- Support comma separated array for group flag in create iamidentitymapping (#3549)
- Add capacity-optimized-prioritized (#3516, #3519)
````

```
## Acknowledgments
Weaveworks would like to sincerely thank:
  @omnibrian, @lloydchang
```

**Comment by Callisto13** [comment]
Hi @lloydchang ! Sorry must have missed your name when I was prepping the notes 🤦‍♀️ .

We don't include the issues in release notes, just the PR, so having 3516 is enough here.

**Review by lloydchang — COMMENTED** [review]
@Callisto13 #3519 is the PR, whereas #3516 is the bug report. Here's an additional commit suggestion for  the release notes. Thank you!

**Pull request merged**